src/main/sensors/gyro.c

```diff
@@ -288,7 +290,8 @@ PG_RESET_TEMPLATE(gyroConfig_t, gyroConfig,
     .yaw_spin_recovery = true,
     .yaw_spin_threshold = 1950,
     .dyn_notch_quality = 70,
-    .dyn_notch_width_percent = 50,
+    .dyn_notch_q_factor = 150,
```

Comment:
I think that setting this higher for default would work better. Same as lowering the dyn_notch_quality

Reply:
Probably around values about 200 would be a nice default. FFT anaylysis is however not very precise sometimes and always delays, so high Q values as with RPM filter are not possible since filter would always miss the peak.
On the topic of quality, I'd rather drop this setting completely but not as part of this PR tho